### ISSUE TITLE
Please provide an Example config file

### ISSUE THREAD

**Issue by markandpathak** [vernemq/vmq-docs#48]
It will be great if an example config file is provided along with documentation for beginners with vernemq. I could not find a reference config file which I can modify as per my requirements.

**Comment by ioolkos** [comment]
@markandpathak good point. I was under the impression we had this, but the page is empty https://docs.vernemq.com/configuring-vernemq/the-vernemq-conf-file
I think I just wanted to paste some "standard" vernemq.conf file there without further comments, as the documentation is in the file itself.
Let me see what I can do.

---
:point_right: Thank you for supporting VerneMQ: https://github.com/sponsors/vernemq
:point_right: Using the binary VerneMQ packages commercially (.deb/.rpm/Docker) requires a paid subscription.
